tests/core/pyspec/eth2spec/test/helpers/phases.py

```diff
@@ -0,0 +1,33 @@
+from .constants import (
```

Comment:
Although we had Phase0, now I wish we could retire "phases" someday since developments are in-parallel.

What do you think about `protocols.py` or `forks.py`?

Reply:
I will go with `forks.py` then. It is consistent with the `fork=...` naming when running `make citest`.
I will not touch the decorators such as `@with_all_phases` for now, as it is out of scope of this PR.